lib/ansible/modules/cloud/azure/azure_rm_storageaccount_facts.py

```diff
@@ -47,16 +47,16 @@
     show_blob_cors:
         description:
             - Show the blob CORS settings for each of the storageaccount's blob.
-            - Note that it will cost a lot time when list all storageaccount rather than querry a single one.
+            - Note that it will cost a lot of time when list all storageaccount rather than query a single one.
```

Comment:
Not sure if this warning applies to querying "all storage accounts" or "all blobs for a storage account". I'm guessing, but correct me if I'm wrong. 

```suggestion
            - Querying all storage accounts will take a long time.
```